src/libstore/include/nix/store/http-binary-cache-store.hh

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
```suggestion
#pragma once
///@file
```

Reply:
And we can land this one write away, just to chip things off to shrink the diff

Reply:
Done in #13851 :)